src/FA2.hs

```diff
@@ -0,0 +1,85 @@
+module FA2 
+  ( tokenMetadataRegistry
+  , addOperator
+  ) where
+
+import Text.Show (Show (..))
+
+import Lorentz
+import Lorentz.Contracts.Spec.FA2Interface hiding (name, sender) 
+
+import Util.Markdown
+import Util.Named
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+
+type Operators = BigMap (Address, Address) ()
```

Comment:
Recently there was [a change to FA2](https://gitlab.com/tzip/tzip/-/merge_requests/85) after which operators are assigned per token_id. You can change the key to `(Address, Address, TokenId)` or have two `Operators` maps (since there are 2 token_ids).

Reply:
As I see in specs the variant `type Operators = BigMap (Address, (Address, TokenId)) ()` suits better, since I can update   map with operatorsParam, that I received as input